### DISCUSSION TITLE
Recapture Recording Limit

### DISCUSSION THREAD

**Discussion by JakeAStrife** [CrankshaftForSteam/crankshaft#19] [Bug Report]
Recapture has an error for recording anything further than 10 minutes. I tried 60 mins, then 30, then 20, 15, and finally only 10 will save. Otherwise I get an error. I definitely have the disk space. Is there a way to fix this?

**Comment by coolavery** [comment]
Sorry about the late reply - there's no hard limit right now, but the plugin is fairly buggy right now with longer recordings.

Last week [someone on Reddit mentioned they were able to do longer recordings](https://old.reddit.com/r/CrankshaftForSteam/comments/wpvooj/had_recapture_working_then_its_gone_all_of_sudden/il00qsh/), maybe something related to updates.

For future reference, can you let me know Steam OS version and update channel you're using (found under Settings -> System)? I can keep that in mind for future reports to see if there's some correlation there.

Sorry that I can't be very helpful with this, to be honest I'm not super familiar with the screen recording/Gstreamer stuff, so I'm still figuring things out.

**Reply by JakeAStrife** [reply]
Thank you for you reply. It's strange how it seems to get the error right after the 12:30 mark. Ive tried uninstalling ReCapture, reinstalling it, and the same with CrankShaft. I also tried a hard reset. The only thing I havent tried is a factory reset... which I'm hesitant to have to do. As one can imagine. 

I apologize for not mentioning the version numbers. I'm a noob at forum style posting.
My SteamOS version is 3.3.1  and I've tried it in Stable and Beta, but it doesn't make a difference, unfortunately. Always 12:30 and then error when I click to end the recording...

No worries. I still have my capture device, I just wanted to be able to record in handheld Game Mode. Using Desktop Mode and OBS tends to make games be buggy and harder to use.
